src/ctapipe/image/cleaning.py

```diff
@@ -544,6 +702,85 @@ def __call__(
         )
 
 
+class LSTImageCleaner(TailcutsImageCleaner):
```

Comment:
Same comment here: a less generic name would be useful, and also please add a refererence to the LST prototype performance paper in the docstring so it's clear where it came from.